### DISCUSSION TITLE
ROLLBACK_COMPLETE Issue in my case

### DISCUSSION THREAD

**Discussion by jeffrey-kor** [mozilla/hubs#4460] [Hubs Cloud Questions] [answered]
Hi, I am following the robin's videos for serving on network. but I got an issue which called ROLLBACK_COMPLETE.

<img width="547" alt="issues" src="https://user-images.githubusercontent.com/46756497/127599227-079d537a-0f61-4af1-a663-f9facafc62d0.png">

How can I solve this problem??

I don't understand what is the point of this issue..

**Comment by pattersonbl2** [comment, marked as answer]
Are you still having issues with this deployment ?

**Reply by jeffrey-kor** [reply]
Nope, I solved this problem. 
In my case, when the rollback_complete error created, I deleted all my vpc instance and then try again.
That's all in my case.

I am reading documents more for learning this.

Current my issue is deploying for enterprise as a customized server.
Thank you for your interests!

**Reply by pattersonbl2** [reply]
No problem, I will marked this issues as answered for now.  We have limited troubleshooting guides when it comes to custom setups.
